### ISSUE TITLE
Some video for firefly about bugs and glitches to fix, but also some useful hints(tricks) for overall playing.

### ISSUE THREAD

**Issue by Rakso69** [Sh0wdown/UnofficialCrusaderPatch#463]
I'm already talking about some SHC tricks/glitches/bugs. I'm going to showcase them on Crusader(game) soon too. Let me know what you think about it, maybe underneath the video in the comments section(so it'll be forever visible, even for future for firefly?)? ;) Help push up the topic, so Firefly might finally take an interest in fixing/removing/altering those, once Warlords will come out. Cheers!
~Rakso
[Youtube video](https://youtu.be/ziFLHqAA0WQ)

**Comment by JuGGerNaunT** [comment]
I like how everybody is trying to use "issue" section as promotion stand

**Comment by Krarilotus** [comment]
> I like how everybody is trying to use "issue" section as promotion stand

i think that shows 2 things:
1. we are in a desperate need for fixing bugs of the game, the video showcases alot of bugs, which apperently are in present in Stronghold Crusader. I know that your mod fixed some issues, that were not adressed by sh0wdown, which are mostly multiplayer related, while sh0wdowns mod fixes so many issues with the AI (which i am really exited and happy about!). 

2. We are in a desperate need to unite Crusader friends on a commen plattform. There is no official plattform from firefly that works. So there are many plattforms popping up, each claming to be the one, but not really all the people know of all of these and they are scattered around the world. Even the chinese community must have their own plattform for the game.

This is how the Firefly Forum looks like. 
http://archive.fireflyworlds.com/index.php?option=com_fireboard&Itemid=271&func=showcat&catid=17
If i try to enter the general section of  Stronghold Cruisader Original, i am promptet with 502 bad gateway. It seems like not a hole lot of care is put into this.

But at least they have a working discord now. And maybe, just maybe, they will notice their huge fanbase about the original 2D titles some day and adress some issues.

Keep up the great work guys and spread the word about the game!

**Comment by J-T-de** [comment]
Yeah, I have to agree with Krarilotus. There is no central point for the Stronghold and Stronghold Crusader community, and this bug-tracker is at the moment the most active thing.

And as long as Sh0wdown and his mods accept 'issues'/posts like this, it is fine I guess. (But obviously, not the perfect solution, but I guess there will be no alternative in the future).
